error importing library using require in Node.js

**What is the current behavior?**
Importing using this code:
```js
var LabelSync = require("github-issues-label-sync");
```

Produces this error:
```sh
Error: Cannot find module 'github-issues-label-sync'
```

**What is the expected behavior?**
This code worked in v1.18.0 😬

--- a/gulpfile.babel.js
+++ b/gulpfile.babel.js
@@ -187,9 +187,11 @@ gulp.task(`setup`, [`clean`], () => {
     });
 });
 
-gulp.task(`postinstall`, [`test:publish`]);
-gulp.task(`clean`, [`clean:docs`, `clean:manual`]);
+
 gulp.task(`test:install`, [`nsp`, `snyk`, `bithound`]);
-gulp.task(`test:publish`, [`test:install`, `package`]);
+gulp.task(`test:publish`, [`package`]);
 gulp.task(`test`, [`lint`]);
+gulp.task(`clean`, [`clean:docs`, `clean:manual`]);
+gulp.task(`postinstall`, [`test:install`]);
+gulp.task(`prepublish`, [`test:publish`]);
 gulp.task(`default`, [`test`]);

